content/version/2/0/code_of_conduct.de.md

```diff
@@ -0,0 +1,137 @@
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmende und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für alle, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Erfahrungsstufe, Bildung, sozialem Status, Nationalität, persönlicher
+Erscheinung, Rasse, Religion oder Geschlechtsidentität und sexueller
+Orientierung.
+
+Wir verpflichten uns, in einer Weise zu handeln und zu interagieren, die zu
+einer offenen, einladenden, vielfältigen, integrativen und gesunden Gemeinschaft
+beiträgt.
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, die zu einem positiven Umfeld für unsere
+Gemeinschaft beitragen, sind:
+
+- Einfühlungsvermögen und Freundlichkeit gegenüber anderen Menschen zeigen
+- Unterschiedliche Meinungen, Standpunkte und Erfahrungen respektieren
+- Konstruktives Feedback geben und würdevoll annehmen
+- Verantwortung übernehmen und uns bei denjenigen entschuldigen, die von unseren
+  Fehlern betroffen sind, und aus den Erfahrungen lernen
+- Konzentration auf das, was nicht nur für uns als Individuen, sondern für die
+  gesamte Gemeinschaft das Beste ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte
+  Versuche sexueller Anbahnung
+- Beleidigende oder abwertende Kommentare, persönliche oder politische Angriffe,
+  Nutzung des Internets für unbilliges Verhalten
+- Öffentliche oder private Belästigungen
+- Das Veröffentlichen von privaten Informationen Anderer, wie zum Beispiel
+  physische oder elektronische Adressen, ohne deren ausdrückliche Erlaubnis
+- Anderes Verhalten, welches in einem professionellen Umfeld begründet als
+  unangemessen betrachtet werden kann
+
+## Verantwortlichkeiten bei der Durchsetzung
+
+Die Gemeinschaftsleitenden sind verantwortlich dafür, unsere Standards für ein
+akzeptables Benehmen klarzustellen und werden angemessen und fair
+korrigierende Maßnahmen ergreifen um auf jegliches Verhalten, das sie für
+unangemessen, bedrohlich oder beleidigend halten, zu reagieren.
+
+Die Gemeinschaftsleitenden haben das Recht und die Verantwortung Kommentare,
+Commits, Code, Wiki-Bearbeitungen, Support-Tickets und andere Beiträge, die
+nicht mit diesem Verhaltenskodex vereinbar sind zu entfernen, zu bearbeiten oder
+abzulehnen, und werden die Gründe für eine Mäßigung mitteilen, wenn es
+angebracht ist.
+
+## Geltungsbereich
+
+Dieser Verhaltenskodex gilt für alle Gemeinschaftsbereiche und gilt auch, wenn
+eine Einzelperson die Gemeinschaft offiziell in öffentlichen Bereichen vertritt.
+Beispiele für die Repräsentation unserer Gemeinschaft sind die Verwendung einer
+offiziellen E-Mail-Adresse, das Posten über ein offizielles Social-Media-Konto
+oder das Auftreten als ernannter Vertreter bei einer Online- oder
```

Comment:
gender neutral: `Vertreter` to `als ernannte Vertretung` also more aligned with `Repräsentation` instead of `Repräsentant` above